diff --git a/src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h b/src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h
index 0e2346463..dc87045a4 100644
--- a/src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h
+++ b/src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h
@@ -40,9 +40,6 @@
 
 #ifndef _WIN32
 #include <sys/time.h>
-#ifdef __GLIBC__
-#include <xlocale.h>
-#endif
 #endif
 
 /// Various utilities for string conversions and date and time manipulation.
