Index: xamarin-pkg-core-setup/src/corehost/cli/fxr/CMakeLists.txt
===================================================================
--- xamarin-pkg-core-setup.orig/src/corehost/cli/fxr/CMakeLists.txt
+++ xamarin-pkg-core-setup/src/corehost/cli/fxr/CMakeLists.txt
@@ -55,4 +55,4 @@ set(HEADERS
 include(../lib.cmake)
 
 install(TARGETS hostfxr DESTINATION corehost)
-install_symbols(hostfxr corehost)
+#install_symbols(hostfxr corehost)
